### PR TITLE
add dashboard for all azure jobs

### PR DESCRIPTION
**Pull request by aramase**
- Add new `sig-azure-on-call` dashboard in `sig-azure` testgrid to see all azure related jobs + jobs running on aks-engine clusters.

/cc @ritazh @PatrickLang

### CODE REVIEW

**Comment by aramase** [comment]
@justaugustus I've swapped all `sig-azure` with `provider-azure` but left `sig-azure` for the testgrid name. 

/cc @ritazh

**Comment by aramase** [comment]
Ah, looks like the prefix name needs to match the group name!

**Comment by aramase** [comment]
@ritazh should we change the `dashboard_group` name to provider-azure too?

**Comment by ritazh** [comment]
I think it should be ok. @feiskyer @andyzhangx WDYT?

**Comment by andyzhangx** [comment]
> I think it should be ok. @feiskyer @andyzhangx WDYT?

ok for me

File: config/testgrids/kubernetes/sig-cloud-provider/azure/config.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 dashboard_groups:
 - name: sig-azure
```
Comment (justaugustus) [inline config/testgrids/kubernetes/sig-cloud-provider/azure/config.yaml]:
```suggestion
- name: provider-azure
```

Comment (aramase) [reply config/testgrids/kubernetes/sig-cloud-provider/azure/config.yaml]: thanks @justaugustus! I've updated the PR.

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml [hunk 2]
```diff
@@ -298,6 +298,6 @@ periodics:
       securityContext:
         privileged: true
   annotations:
-    testgrid-dashboards: sig-windows
+    testgrid-dashboards: sig-windows, provider-azure-on-call
```
Comment (PatrickLang) [inline config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: can you skip the staging job? that's used to test new things before they're added to the above jobs.

Comment (ritazh) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: Can you pls also remove the testgrid email alert for the staging job? Thx!

Comment (aramase) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: sure!

Comment (aramase) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: Removed the testgrid email alert for staging job.

@ritazh PTAL. Thanks!

**Comment by justaugustus** [comment]
/lgtm
/approve

@krzyzacy @wojtek-t -- can you approve for `config/`?

**Comment by aramase** [comment]
ping @krzyzacy @wojtek-t

**Comment by cblecker** [comment]
/unassign @krzyzacy @wojtek-t 

/assign @PatrickLang @adelina-t @bclau 
for sig-windows

/assign @bowei @cadmuxe @jingax10 @rramkumar1 @mrhohn
for sig-network

The tests that are not approved are owned by sig-windows and sig-network. Please ping those owners for approval, as they are responsible for their tests :)

**Comment by cblecker** [comment]
/unassign @bowei @cadmuxe @jingax10 @rramkumar1 @MrHohn

Thanks @MrHohn !

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml [hunk 3]
```diff
@@ -305,5 +305,4 @@ periodics:
   annotations:
     testgrid-dashboards: sig-windows
     testgrid-tab-name: aks-engine-azure-windows-master-staging
-    testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
```
Comment (PatrickLang) [inline config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: thanks - suggested this earlier

**Comment by PatrickLang** [comment]
/approve

thanks!

**Pull request merged**